Jai0212/Cash-App-Bias-Busters
# 115 create tests for frontend components

# Description
This pull request introduces a series of test cases to ensure the functionality and reliability of various frontend components using Jest and React Testing Library, as mentioned in the issue #115. The following components and pages have been tested:

**Test Cases Added:**
- Control Buttons
- Dashboard
- Demographics Selector
- Graph Data Default
- QR Code Share
- Slider
- Tour Guide
- Chart Component 2
- Dashboard 2
- Control Buttons 2


# Test Results:
- **Test Suites:** 22 test suites ran successfully.
- **Tests:** 62 tests ran successfully.


# How to Test:
1. Clone this branch `115-create-tests-for-frontend-components-new`.
2. Run the tests using `npx jest --coverage` to ensure all tests pass and coverage is up to date.
